src/components/viewmanager/viewManager.js

```diff
@@ -0,0 +1,183 @@
+define(['viewcontainer', 'focusManager', 'queryString', 'layoutManager'], function (viewcontainer, focusManager, queryString, layoutManager) {
```

Comment:
is this really a new file, or just mis-behaving rename + modification?.. skipping review of this file until clarified.

Reply:
I fixed this issue, please check again